gcn_email_service/__init__.py

```diff
@@ -100,7 +105,15 @@ def recieve_alerts(consumer):
 @on_exception(expo, RateLimitException)
 @limits(calls=MAX_SENDS, period=1)
 def send_raw_ses_message_to_recipient(client, message, recipient):
-    BODY_TEXT = str(email.message_from_bytes(message.value()))
+    BODY_TEXT = ""
+    file = tempfile.NamedTemporaryFile()
```

Comment:
You're leaking files and file descriptors. Generally, temporary files should be used in a `with:` statement so that they are deleted automatically:

```python
with tempfile.NamedTemporaryFile() as f:
    ...
```

Also, I strongly suspect that we can do this without any files at all, strictly in memory.

Reply:
Yea, I think I can remove this completely, I made changes so it is not needed